packages/examples/packages/background-events/package.json

```
@@ -0,0 +1,74 @@
+{
+  "name": "@metamask/background-events-example-snap",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.0",
```
Needs to start as 0.0.0